tests/test_processing_common.py

```diff
@@ -60,7 +60,10 @@ def get_component(self, attribute, **kwargs):
             component_class_name = component_class_name[0]
 
         component_class = processor_class_from_name(component_class_name)
-        component = component_class.from_pretrained(self.tmpdirname, **kwargs)  # noqa
+        try:
+            component = component_class.from_pretrained(self.tmpdirname, **kwargs)  # noqa
+        except AttributeError:
+            component = component_class.from_pretrained(self.model_id, **kwargs)  # noqa
```

Comment:
Why do we need the try/except patten here?

Reply:
because some processors are initialized with a self.tmpdirname, some don't have this attribute but have a self.model_id  instead - I can check for attribute presence/absence instead of try/except, it'd be cleaner